include/aos/common/crypto/mbedtls/mbedtls_config.h

```diff
@@ -9,7 +9,19 @@
  */
```

Comment:
I propose to merge this configuration with our config more accurately, because if we just copy past we can remove necessary in the future configuration from 3.6, for example MBEDTLS_PLATFORM_GMTIME_R_ALT etc

Reply:
fixed